f/flash-b.md

```diff
@@ -0,0 +1,10 @@
+# Flash-b
+
+
+![dan](flash.jpg)
+
+Anne Flaccida Randrianirina known as Flash-B has been keen on music since she was a kid. When she was 17 years old, she used to spend her time in her father's studio.She became famous in 2018 when she started her first song entitled 'Fosy Jokera'.
```

Comment:
Please use spaces after full-stops or periods (.).

Reply:
hello,

I have fixed the issue. How is it, now?